docs/content/operations/multitenancy.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc_page
+---
+# Multitenancy Considerations
+
+Druid is often used to power user-facing data applications and has several features built in to better support high 
+volumes of concurrent queries.
+
+## Parallelization Model
+
+Druid's fundamental unit of computation is a [segment](../design/segments.html). Nodes scan segments in parallel and a 
+given node can scan `MIN(druid.processing.numThreads, Runtime.getRuntime().availableProcessors())` concurrently. To 
```

Comment:
Another question I haven't found a good answer to is how this applies to cgroups. If someone has the jvm running in a limited cgroup, I'm not sure if the reported number of processors equals the number of cgroup CPUs the jvm has access to.